src/parser.js

```diff
@@ -0,0 +1,59 @@
+require('dotenv').config();
```

Comment:
Let's rename this file `src/parser.js` to something less generic.  How about `src/wiki-feed-parser.js`?  We have a lot of "parsers" now, so it's confusing.